zerver/views/message_fetch.py

```diff
@@ -135,7 +137,8 @@ class NarrowBuilder:
     def __init__(self, user_profile: UserProfile, msg_id_column: str) -> None:
         self.user_profile = user_profile
         self.msg_id_column = msg_id_column
-        self.user_realm = user_profile.realm
+        # if user_profile is not None:
+            # self.user_realm = user_profile.realm
```

Comment:
I imagine we'll just want to make `realm` an additional parameter here?

Reply:
Yes